.github/workflows/create-tag.yml

```diff
@@ -0,0 +1,21 @@
+# This action creates a release the first Monday each month at 5:00 UTC.
+name: "Create and push release tag"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 5 1-7 * 1"
```

Comment:
Is this "first Monday of the month"?

Reply:
Whoops, yeah I messed that up. Maybe a condition here is slightly better like we do in composer or osbuild